### PR TITLE
add: `waynergy`

### PR DESCRIPTION
**Pull request by erenfro**
Adds waynergy to pacstall packages. A synergy, barrier, input-leap compatible waynergy-specific client.

### CODE REVIEW

File: packages/waynergy/waynergy.pacscript [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "gnome-session: Idle inhibition for GNOME"
+)
+maintainer=("Eric Renfro <psi-jack@linux-help.org>")
+incompatible=("ubuntu:bionic" "ubuntu:focal" "ubuntu:kinetic" "debian:stretch" "debian:buster" "debian:bullseye")
+source=("${pkgname}-${pkgver}.tar.gz::https://github.com/r-c-f/waynergy/archive/refs/tags/v${pkgver}.tar.gz")
+sha256sums=("b8b2d12588cabfe161ce21a5cd898f06bdfd55c0106ecd4a26c8628aafeff990")
+build() {
+  cd "${_archive}"
+  meson build --prefix="/usr"
+  ninja -C build
```
Comment (vigress8) [inline packages/waynergy/waynergy.pacscript:21]:
```suggestion
  ninja -C build -j"${NCPU}"
```
To utilize all CPU threads for the build.

File: packages/waynergy/waynergy.pacscript [hunk 2]
```diff
@@ -0,0 +1,28 @@
+pkgname=waynergy
+pkgver=0.0.17
+pkgrel=3
```
Comment (vigress8) [inline packages/waynergy/waynergy.pacscript:3]:
```suggestion
```

**Review by vigress8 — CHANGES_REQUESTED** [review]

**Comment by Elsie19** [comment]
Fix the title to conform to our standards, as explained in [our docs](https://github.com/pacstall/pacstall/wiki/101.5-Maintaining-a-Package#updating-your-package). You can also look at [past PRs](https://github.com/pacstall/pacstall-programs/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed+label%3A%22Update+Package%22) to get an idea of how to format the title if you're unsure.

---

You gotta play by the rules

**Comment by erenfro** [comment]
> Fix the title to conform to our standards, as explained in [our docs](https://github.com/pacstall/pacstall/wiki/101.5-Maintaining-a-Package#updating-your-package). You can also look at [past PRs](https://github.com/pacstall/pacstall-programs/pulls?q=is%3Apr+sort%3Aupdated-desc+is%3Aclosed+label%3A%22Update+Package%22) to get an idea of how to format the title if you're unsure.

All I'm seeing here is for updated packages, not for new packages being added. So this is not really entirely clear.

**Review by oklopfer — APPROVED** [review]

**Pull request merged**